Functions/It.ps1

```diff
@@ -154,25 +154,25 @@ function ItImpl
     }
 
     #the function is called with Pending or Skipped set the script block if needed
-    if ($null -eq $test) { $test = {} }
+    if ($null -eq $Test) { $Test = {} }
 
     #mark empty Its as Pending
     if ($PSVersionTable.PSVersion.Major -le 2 -and
         $PSCmdlet.ParameterSetName -eq 'Normal' -and
-        [String]::IsNullOrEmpty((Remove-Comments $test.ToString()) -replace "\s"))
+        [String]::IsNullOrEmpty((Remove-Comments $Test.ToString()) -replace "\s"))
     {
         $Pending = $true
     }
     elseIf ($PSVersionTable.PSVersion.Major -gt 2)
     {
         #[String]::IsNullOrWhitespace is not available in .NET version used with PowerShell 2
         # AST is not available also
-        $testIsEmpty =
-            [String]::IsNullOrEmpty($test.Ast.BeginBlock.Statements) -and
-            [String]::IsNullOrEmpty($test.Ast.ProcessBlock.Statements) -and
-            [String]::IsNullOrEmpty($test.Ast.EndBlock.Statements)
+        $TestIsEmpty =
```

Comment:
Please revert this, and the usages. $testIsEmpty is a "local" variable, and those should be lowercase.